snapshots/happiness/2024-06-09/happiness.xls.dvc

```diff
@@ -0,0 +1,30 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: World Happiness Report
+    description: |-
+      The World Happiness Report is a partnership of Gallup, the Oxford Wellbeing Research Centre, the UN Sustainable Development Solutions Network, and the WHR’s Editorial Board.
+      It reviews the state of happiness in the world today and shows how the science of happiness explains personal and national variations in happiness.
+    date_published: "2024-03-08"
+    version_producer: 2024
+
+    # Citation
+    producer: "University of Oxford: Wellbeing Research Centre"
+    citation_full: |-
+      Helliwell, J. F., Layard, R., Sachs, J. D., De Neve, J.-E., Aknin, L. B., & Wang, S. (Eds.). (2024). World Happiness Report 2024. University of Oxford: Wellbeing Research Centre.
+
+    # Files
+    url_main: https://worldhappiness.report/ed/2024/
+    url_download: https://happiness-report.s3.amazonaws.com/2024/DataForFigure2.1+with+sub+bars+2024.xls
+    date_accessed: 2024-06-09
+
+    # License
+    license:
+      name: ""
```

Comment:
I did struggle with that because on the website there is no license mentioned or explicit instructions for how to (re-)publish the data. THe closest thing would be the citation and isbn on the report page (https://worldhappiness.report/ed/2024/). I added this now as license url - let me know if that works also!